server/jersey-server/src/main/java/com/quorum/tessera/server/monitoring/MetricsResource.java

```diff
@@ -39,7 +39,7 @@ public Response getMetrics() {
 
     return Response.status(Response.Status.OK)
         .header(HttpHeaders.CONTENT_TYPE, TEXT_PLAIN)
-        .entity(formattedMetrics.toString().trim())
+        .entity(formattedMetrics.toString())
```

Comment:
@chris-j-h - made the change you suggested. Are you able to help with the build issues?

Reply:
```
[@octokit/rest] `const Octokit = require("@octokit/rest")` is deprecated. Use `const { Octokit } = require("@octokit/rest")` instead
Error:  Missing Slack Incoming Webhooks URL.
      Please configure "SLACK_WEBHOOK" as environment variable or
      specify the key called "url" in "with" section.
```

Reply:
I've disabled the slack notification step for the time being so this can be merged while we debug - i also moved the newline formatting into the PrometheusProtocolFormatter so that we can add some unit tests